src/content/ru/2020/capabilities.md

```diff
@@ -0,0 +1,330 @@
+---
+part_number: II
+chapter_number: 13
+title: Возможности
+description: Глава "Возможности" издания Web Almanac за 2020 год посвящена совершенно новым, мощным API в веб-платформе.
+authors: [christianliebel]
+reviewers: [tomayac]
+analysts: [tomayac]
+translators: [lex111]
+discuss: 2049
+results: https://docs.google.com/spreadsheets/d/1N6j1qKv7vc51p1W9z_RrbJX9Se3Pmb-Uersfz4gWqqs/
+queries: 13_Capabilities
+christianliebel_bio: Кристиан Либел работает консультантом в компании Thinktecture, помогая клиентам из различных сфер бизнеса создавать качественные веб-приложения. Он обладатель награды Microsoft MVP в области технологий для разработчиков, Google GDE для веба/возможностей и Angular, а также участвует в рабочей группе W3C по веб-приложениям.
+featured_quote: Состояние веб-возможностей в 2020 году хорошее, поскольку новые и мощные API регулярно появляются в новых версия Chromium-браузеров. Некоторые API уже доступны и в других браузерах.
+featured_stat_1: 0.0006%
+featured_stat_label_1: (Chrome) Страница загружается с помощью Async Clipboard API
+featured_stat_2: 0.49%
+featured_stat_label_2: Сайты, использующие Storage Manager API
+featured_stat_3: 363
+featured_stat_label_3: Сайты, позволяющие устанавливать связанные с ним приложения
+---
+
+## Введение
+
+[Прогрессивные веб-приложения (Progressive Web Apps, PWA)](./pwa) — это кроссплатформенная модель приложения, основанная на веб-технологиях. С помощью сервис-воркеров такие приложения работают даже в офлайн-режиме. [Манифест веб-приложений](https://developer.mozilla.org/en-US/docs/Web/Manifest) позволяет пользователям добавлять PWA на главный экран или в список программ смартфона. При последующем открытии PWA выглядит как нативное приложение. Тем не менее, PWA могут использовать только функциональность и возможности, доступные через API веб-платформы. Какими-либо нативными интерфейсами не получится воспользоваться и это создаёт пропасть между нативными и веб-приложениями.
```

Comment:
Use Russia resources where available.

Also do you think the `lang="en"` attributes are needed? It would mean converting from markdown to HMTL. This is a general question throughout the chapter. Technical terms may be OK things like "Capabilities Project" probably should have them shouldn't they?

```suggestion
<a href="./pwa">Прогрессивные веб-приложения (<span lang="en">Progressive Web Apps</span>, PWA)</a> — это кроссплатформенная модель приложения, основанная на веб-технологиях. С помощью сервис-воркеров такие приложения работают даже в офлайн-режиме. [Манифест веб-приложений](https://developer.mozilla.org/ru/docs/Web/Manifest) позволяет пользователям добавлять PWA на главный экран или в список программ смартфона. При последующем открытии PWA выглядит как нативное приложение. Тем не менее, PWA могут использовать только функциональность и возможности, доступные через API веб-платформы. Какими-либо нативными интерфейсами не получится воспользоваться и это создаёт пропасть между нативными и веб-приложениями.
```